tests/env_test.py

```diff
@@ -0,0 +1,78 @@
+import threading
+import time
+from typing import List
+
+
+class Msg:
```

Comment:
This class is called `Message` in blpapi (just to be consistent with the original blpapi)